lib/plugins/create/templates/hello-world/serverless.yml

```diff
@@ -21,4 +21,3 @@ functions:
       - http:
           path: hello-world
           method: get
-          cors: true
```

Comment:
This was added by @DavidWells so that CORS is always in place and it's easier to start with the `hello-world` template.